webapp/tests/mocha/unit/initial-replication.spec.js

```diff
@@ -169,6 +167,7 @@ describe('Initial replication', () => {
         bulkGet: sinon.stub(),
       };
       clock = sinon.useFakeTimers();
+      initialReplication = rewire('../../../src/js/bootstrapper/initial-replication');
```

Comment:
is this needed because of the `initialReplication.__set__('displayTooManyDocsWarning', displayTooManyDocsWarning);` on line 658 ?

Reply:
Yes. The alternative is to just use require and add a setter to the production code, which is the way we used to do things in the good old days  :)